06.bowling_object/frame.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class Frame
+  def initialize(frame_number)
+    @frame_number = frame_number
+    @shots = []
+    @bonus_shots = []
+  end
+
+  def record_shot(shot_score)
+    score = shot_score == 'X' ? 10 : shot_score.to_i
```

Comment:
```suggestion
  def record_shot(shot)
    shot = shot == 'X' ? 10 : shot.to_i
```

なるべく引数とローカル変数は同じ名前にしない方がいいですが、こういうケースは許容範囲かなーと思います。